src/Microsoft.TemplateEngine.Core/Include.cs

```diff
@@ -8,11 +8,12 @@ namespace Microsoft.TemplateEngine.Core
 {
     public class Include : IOperationProvider
     {
-        public Include(string startToken, string endToken, Func<string, Stream> sourceStreamOpener)
+        public Include(string startToken, string endToken, Func<string, Stream> sourceStreamOpener, string id = null)
```

Comment:
Since these are all constructed from a single codepath, defaults and/or overloads aren't preferred